studio/src/pages/RequestorPage/types.ts

```diff
@@ -47,7 +47,7 @@ export const ProbedRouteSchema = z.object({
   currentlyRegistered: z.boolean(),
   registrationOrder: z.number().default(-1),
   routeOrigin: z.enum(["discovered", "custom", "open_api"]),
-  openApiSpec: z.string().optional(),
+  openApiSpec: z.string().nullish().optional(),
```

Comment:
This the second definition of the `app route` we should investigate if we can delete one of them. But given the deadline, i haven't done this for this PR